std/net/curl.d

```diff
@@ -776,13 +776,13 @@ if ( isCurlConn!Conn && (is(T == char) || is(T == ubyte)) )
  *
  * See_Also: $(LREF HTTP.Method)
```

Comment:
Please add a "Returns:" clause as in the other functions returning a value. E.g.:

```
 * Returns:
 * A `T[]` range containing the response body for this request (`null` in case of FTP).
```